sonar-db/src/main/java/org/sonar/db/MyBatis.java

```diff
@@ -236,7 +234,7 @@ public MyBatis start() {
       UserMapper.class, GroupMapper.class, UserGroupMapper.class, UserTokenMapper.class,
       FileSourceMapper.class,
       NotificationQueueMapper.class,
-      GroupMembershipMapper.class, QualityProfileMapper.class, ActiveRuleMapper.class,
+      GroupMembershipMapper.class, QualityProfileMapper.class, ActiveRuleMapper.class, QProfileChangeMapper.class,
```

Comment:
would be better to have new mapper on a dedicated line, more diff and conflict friendly